### PR TITLE
JNG-4922 Set the default attributes of entity to false

### PR DESCRIPTION
**Pull request by Madmat8**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4922" title="JNG-4922" target="_blank">BugJNG-4922</a>  Create Transfer Object with required fields maps an Entity with fields with default values throws ValidationException
  </td></table>
  <br />

### CODE REVIEW

File: judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl [hunk 1]
```diff
@@ -89,7 +89,8 @@ rule CreateMappedTransferAttributeEntityDefault
         and s.getterExpr.features.first.member.defaultExpr.isDefined() and not JSL!TransferDefault.all().exists(d | d.member.reference == s)
       t.name = defaultDefaultNamePrefix + s.name + defaultDefaultNameMidfix + s.getterExpr.features.first.member.eContainer.name + defaultDefaultNamePostfix;
       t.setId("(jsl/" + s.getId() + ")/CreateMappedTransferAttributeEntityDefault");      
-      t.binding = s.getterExpr.features.first.member.defaultExpr.equivalent("CreateDefaultValueForPrimitiveEntityMember");               
+      t.binding = s.getterExpr.features.first.member.defaultExpr.equivalent("CreateDefaultValueForPrimitiveEntityMember");
+      t.required = false;
```
Comment (bencegelei) [inline judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl:93]: This seems to override the original behaviour in `AbstractCreateTransferAttributeForTransferFieldDeclaration` which is `t.required = s.isRequired`. Is it really the intended solution? Wouldn't a proper solution for [JNG-4942](https://blackbelt.atlassian.net/browse/JNG-4942) - by modifying validation procedures - could solve this as well as the other cases?

Comment (robertcsakany) [reply judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl:93]: I agree

Comment (robertcsakany) [reply judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl:93]: I agree

**Review by robertcsakany — CHANGES_REQUESTED** [review]

File: judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl (hunk 1 above)
Comment (robertcsakany) [reply judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl:93]: I agree.

**Review by robertcsakany — CHANGES_REQUESTED** [review]

File: judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl (hunk 1 above)
Comment (Madmat8) [reply judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferDeclarationTransferAttribute.etl:93]: [49e7d9d](https://github.com/BlackBeltTechnology/judo-tatami-jsl/pull/187/commits/49e7d9d7a81aaeb67d04d9a399e7f2533f949213)

**Review by robertcsakany — APPROVED** [review]

**Review by bencegelei — APPROVED** [review]

**Pull request merged**